packages/marketplace/contracts/libraries/LibAsset.sol

```diff
@@ -159,19 +152,13 @@ library LibAsset {
 
     /// @dev function to verify if the order is a bundle and validate the bundle price
     /// @param leftAsset The left asset.
-    /// @param rightAsset The right asset.
-    function verifyPriceDistribution(Asset memory leftAsset, Asset memory rightAsset) internal pure {
+    /// @param fill The fill.
+    function verifyPriceDistribution(Asset memory leftAsset, LibOrder.FillResult memory fill) internal pure {
```

Comment:
We should not import LibOrder in LibAsset, those should be independent. Lets just pass `uint256` to the `verifyPriceDistribution` function.